Se crea un proyecto test para practicar

Jesus-Vila opened the pull request:
- Se crea el proyecto
- Se agrega la configuración de tox

kyokun committed:
Archivos para ejectar tox

vila8 committed:
Initial config

kyokun commented on products/.idea/Test.iml:
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Este archivo que hace aquí?

kyokun commented on products/.idea/misc.xml:
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Este igual

kyokun commented on products/.idea/modules.xml:
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Este tampoco

kyokun commented on products/.idea/workspace.xml:
```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Ni este

kyokun commented on products/api/models/__init__.py:
```diff
@@ -0,0 +1 @@
+from .static import *
```
Falta linea en blanco al final del archivo

kyokun commented on products/api/models/static.py:
```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+""" Static models """
```
De ser: `"""Static models."""`

kyokun replied:
No debe haber espacios y debe terminar en `.`

kyokun commented on products/api/models/static.py:
```diff
@@ -0,0 +1,25 @@
+
+
+class Industry(models.Model):
+    name = models.CharField(max_length=32)
```
Hay que agregar doctrings en cada clase

kyokun commented on products/api/serializers/static.py:
```diff
@@ -0,0 +1,30 @@
+from api.models import Country, Category, Industry, State
+
+
+class IndustrySerializer(serializers.ModelSerializer):
```
Aui también te faltan docstrings

kyokun commented on products/api/views/static.py:
```diff
@@ -0,0 +1,28 @@
+                             StateSerializer, CategorySerializer)
+
+
+class IndustryViewSet(viewsets.ModelViewSet):
```
Igual docstrings

vila8 committed:
Remove .idea folder, add docstrings and add new line

Add docstrings to products/api/*/statics.py files.
Add new line to products/api/models/__init__.py file.

kyokun commented on products/api/serializers/static.py:
```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+""" Static serializers. """
```
Te faltó corregir este doctring

kyokun commented on products/api/urls.py:
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+""" API urls """
```
Te faltó corregir este doctring

kyokun commented on products/api/views/static.py:
```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+""" Static views. """
```
Te faltó corregir este doctring

vila8 committed:
Edit docstrings
